src/geometry/CTiglIntersectionCalculation.h

```diff
@@ -99,6 +112,9 @@ class CTiglIntersectionCalculation
     // gives a reference to the computed wire
     TIGL_EXPORT TopoDS_Wire GetWire(int wireID);
 
+    // gives a reference to the computed vertex
+    TIGL_EXPORT TopoDS_Vertex GetVertex(int vertexID);
```

Comment:
Is there a reason, why you return a vertex and not directly a point? I just ask, because you are always converting the vertex later on into a point.